domains/julian.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "This (will be) is my portfolio",
+    "repo": "https://github.com/0x1AE7F/portfolio.github.io",
+    "owner": {
+        "username": "0x1AE7F",
+        "email": ""
+        "discord": "0x1AE7F#5205"
+    },
+    "record": {
+        "CNAME": "0x1ae7f.github.io"
```

Comment:
404?

Reply:
My bad.. Its fixed now

Reply:
@0x1AE7F Please add more content to your website.